aws_lambda_scripts/requirements.txt

```diff
@@ -4,6 +4,7 @@ certifi==2024.6.2
 cffi==1.16.0
 charset-normalizer==3.3.2
 cryptography==42.0.8
+github-api-toolkit @ git+https://github.com/ONS-Innovation/github-api-package.git
```

Comment:
We should pin this to a sepcific release. I have opened an issue on the toolkit repo, we can retrospectively change this once the release is built (I'll open a JIRA).

Reply:
Jira ticket added, will complete in next sprint